api/user/posts/model.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+
+from api.app import db
+
+
+class Post(db.Model):
+    _id = db.Column(db.Integer, primary_key=True)
+    _published = db.Column("published", db.DateTime, nullable=False)
+    _title = db.Column("title", db.String(120), nullable=False)
+    _origin = db.Column("origin", db.Text, nullable=False)
+    _source = db.Column("source", db.Text, nullable=False)
+    _description = db.Column("short_desc", db.String(100))
+    _contentType = db.Column("contentType", db.String(50), nullable=False)
+    _content = db.Column("content", db.Text, nullable=False)
+
+    # categories will be comma separated values
+    _categories = db.Column("categories", db.Text)
+
+    # 0 -> "PUBLIC", 1-> "FRIENDS"
+    _visibility = db.Column("visibility", db.Boolean, nullable=False)
+
+    # 0 -> False, 1 -> True
+    _unlisted = db.Column("unlisted", db.Boolean, nullable=False)
+
+    # Foreign Key
+    _auth_id = db.Column("auth_id", db.Integer, db.ForeignKey("author.id"), nullable=False)
```

Comment:
```suggestion
    _author_id = db.Column("author_id", db.Integer, db.ForeignKey("author.id"), nullable=False)
```
at least one person somewhere, once will confuse this with authentication oitherwise

Reply:
👍🏻